src/common/ITEMS/classic/others.ts

```diff
@@ -0,0 +1,6 @@
+import { itemIndexableList } from 'common/ITEMS/Item';
```

Comment:
I left this in more as a "this is how you can add additional items to classic". We can remove the other item list if anyone feels strongly about it, but I felt that leaving this here sets up a pattern for others to follow.